engine.py

```diff
@@ -408,7 +408,7 @@ def update_image():
             base, ext = render_output.rsplit('.', 1)
             # denoise data has the name .denoise.exr
             denoise_options = "-t%d" % self.rm.threads
-            denoise_data = base + '.denoise.' + 'exr'
+            denoise_data = base + '.variance.' + 'exr'
```

Comment:
the denoise multilayer EXR was renamed to .variance. So this is to maintain internal denoising ability